lib/src/sdk.ts

```diff
@@ -66,9 +65,39 @@ const ExchangeType = {
   SWAP_NG: "SWAP_NG",
 } as const;
 
+
+type CustomLogHandlersType = {
+  "custom.throwExchangeErrorToLedgerLive": (_: ExchangeError) => void;
+  "custom.throwGenericErrorToLedgerLive": (_: Error) => void;
+};
+
+class CustomErrorModule extends CustomModule {
+  throwExchangeErrorToLedgerLive(error: ExchangeError) {
+    return this.request<
+      Parameters<
+        CustomLogHandlersType["custom.throwExchangeErrorToLedgerLive"]
+      >[0],
+      ReturnType<CustomLogHandlersType["custom.throwExchangeErrorToLedgerLive"]>
+    >("custom.throwExchangeErrorToLedgerLive", error);
+  }
+  throwGenericErrorToLedgerLive(error: Error) {
+    return this.request<
+      Parameters<
+        CustomLogHandlersType["custom.throwGenericErrorToLedgerLive"]
+      >[0],
+      ReturnType<CustomLogHandlersType["custom.throwGenericErrorToLedgerLive"]>
+    >("custom.throwGenericErrorToLedgerLive", error);
+  }
+}
+
+/**
+ * Should match our swap live apps path naming  :/
+ */
 function getCustomModule(client: WalletAPIClient) {
   return {
     exchange: new ExchangeModule(client),
+    swap: new CustomErrorModule(client), 
```

Comment:
Do we must move this Module to a different file as it is swap specific? (now, I am thinking...I am not sure if we are in Moonpay context that we will have access to this drawer)